app/src/main/java/space/taran/arkretouch/presentation/edit/EditScreen.kt

```diff
@@ -850,3 +890,237 @@ private fun ExitDialog(
         }
     )
 }
+
+@Composable
+fun NewImageOptions(
+    defaultResolution: Resolution,
+    maxResolution: IntSize,
+    _backgroundColor: Color,
+    navigateBack: () -> Unit,
+    editManager: EditManager,
+    persistDefaults: (Color, IntSize) -> Unit,
+    onConfirm: () -> Unit
+) {
```

Comment:
Maybe extract dialog to separate file?
And replace `maxResolution: IntSize` with `Resolution`